ref/provided-images.md

```diff
@@ -1,36 +1,32 @@
 
 Anbox Cloud images are regular [Ubuntu cloud images](https://cloud-images.ubuntu.com/) where Anbox Cloud and its dependencies are installed. Unnecessary packages are removed to improve images size, boot time and security.
 
-## Official images
-
-The following table lists all images that are available on the official image server, along with their corresponding Android and Ubuntu versions. Please note the support status for each of the images.
-
-| Name                        | Android Version | Ubuntu Version | Support Status     | Available since |
-|-----------------------------|-----------------|----------------|------------|---------------|
-| `jammy:android13:amd64`     | 13              | 22.04          | supported | 1.16 |
-| `jammy:android13:arm64`     | 13              | 22.04          | supported | 1.16 |
-| `jammy:android12:amd64`     | 12              | 22.04          | supported | 1.14 |
-| `jammy:android12:arm64`     | 12              | 22.04          | supported | 1.14 |
-| `jammy:android11:amd64`     | 11              | 22.04          | supported | 1.14 |
-| `jammy:android11:arm64`     | 11              | 22.04          | supported | 1.14 |
-| `jammy:android10:amd64`     | 10              | 22.04          | supported | 1.14 |
-| `jammy:android10:arm64`     | 10              | 22.04          | supported | 1.14 |
-| `bionic:android12:amd64`    | 12              | 18.04          | deprecated as of 1.16 | 1.12 |
-| `bionic:android12:arm64`    | 12              | 18.04          | deprecated as of 1.16 | 1.12 |
-| `bionic:android11:amd64`    | 11              | 18.04          | deprecated as of 1.16 | 1.10 |
-| `bionic:android11:arm64`    | 11              | 18.04          | deprecated as of 1.16 | 1.10 |
-| `bionic:android10:amd64`    | 10              | 18.04          | deprecated as of 1.16 | 1.0  |
-| `bionic:android10:arm64`    | 10              | 18.04          | deprecated as of 1.16  | 1.0  |
-| `bionic:android7:amd64`     | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
-| `bionic:android7:arm64`     | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
+## Supported images
+
+The following table lists supported images available on the official image server, along with their corresponding Android and Ubuntu versions.
+
+| Name                        | Android Version | Ubuntu Version | Available since |
+|-----------------------------|-----------------|----------------|---------------|
+| `jammy:android13:amd64`     | 13              | 22.04          | 1.16 |
+| `jammy:android13:arm64`     | 13              | 22.04          | 1.16 |
+| `jammy:android12:amd64`     | 12              | 22.04          | 1.14 |
+| `jammy:android12:arm64`     | 12              | 22.04          | 1.14 |
+
+## Deprecated images
+The following table lists the images that are available on the official image server but are deprecated. They will become unsupported after two releases.
```

Comment:
Should we also add _when_ they were deprecated? If somebody is reading this after 1.19 is released, how would they know if it was deprecated in 1.18 or 1.19?

Reply:
Added a deprecated since column which would address this concern.